scss/application.scss

```diff
@@ -0,0 +1,52 @@
+@import url(http://fonts.googleapis.com/css?family=Roboto);
+
+// Colors
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
+
+// Sizes
+$nav-height: 90px;
+$footer-height: 30vh;
+$triangle-heigth: 90px;
+$triangle-width: 100px;
+$tech-logo-size: 100px;
+
+// Font sizes
+$font-big: 24px;
+$font-medium: 16px;
+$font-small: 12px;
+
+body {
+  margin: 0px 0px 0px 0px;
+  overflow: hidden;
```

Comment:
The equalizer is supposed to resolve this, you should be fine without this rules :smile_cat: